tests/test_job.py

```diff
@@ -500,3 +500,20 @@ def test_job_resubmit_all(hq_env: HqEnv):
 
     table = hq_env.command(["resubmit", "1"], as_table=True)
     assert table[3] == ["Tasks", "3; Ids: 2, 7, 9"]
+
+
+def test_job_wait(hq_env: HqEnv):
+    hq_env.start_server()
+    hq_env.start_worker()
+    hq_env.command(["submit", "sleep", "1"])
+    r = hq_env.command(["wait", "1"])
+    assert "Waiting for 1 job(s)" in r
```

Comment:
Please add a check that the job has actually finished after the wait command ended.